packages/fw-avatar/web-test-runner.config.js

```diff
@@ -0,0 +1,45 @@
+import { legacyPlugin } from "@web/dev-server-legacy";
+import { playwrightLauncher } from "@web/test-runner-playwright";
+import { summaryReporter } from '@web/test-runner';
+
+const mode = process.env.MODE || "dev";
+if (!["dev", "prod"].includes(mode)) {
+  throw new Error(`MODE must be "dev" or "prod", was "${mode}"`);
+}
+
+const browsers = {
+  chromium: playwrightLauncher({ product: "chromium" }),
+  firefox: playwrightLauncher({ product: "firefox" }),
+  webkit: playwrightLauncher({ product: "webkit" }),
+};
+
+const noBrowser = (b) => {
+  throw new Error(`No browser configured named '${b}'; using defaults`);
+};
+let commandLineBrowsers;
+try {
+  commandLineBrowsers = process.env.BROWSERS?.split(",").map(
+    (b) => browsers[b] ?? noBrowser(b)
+  );
+} catch (e) {
+  console.warn(e);
+}
+
+const configObj = {
+  rootDir: ".",
+  files: ["./**/*test.js"],
+  nodeResolve: { exportConditions: mode === "dev" ? ["development"] : [] },
```

Comment:
What does `nodeResolve` actually do?

Reply:
nodeResolve is to Resolve bare module imports using node resolution. [link](https://modern-web.dev/docs/test-runner/cli-and-configuration/).
Without node-resolve the test command would give error about not providing relative path of node modules libraries imported as bare modules.
I followed the [starter kit](https://github.com/lit/lit-element-starter-js/blob/main/web-test-runner.config.js) using web-test-runner , to setup the config file.

Also the flag `--node-resolve` is `true` in the script for `npm test`, so even if i remove it from config.js file it should work fine. Should i remove it?

Reply:
Better to remove the node-resolve and the test files path from here, and just have it in the config file